File: College/Assignments/ACPL/CPP/3.cpp
#include<iostream>
using namespace std;

class Solution{
public:
    int fibonacci(int n){
        if(n < 0){
            cout<<"Invalid value of n";
            return -1;
        }
        if(n == 0 || n == 1) return 1;
        return fibonacci(n-1) + fibonacci(n-2);
    }
};

int main(){
    Solution sol;
    cout<<sol.fibonacci(5);
    return 0;
}

File: College/Assignments/ACPL/Python/3.py
def fibonacci(n: int) -> int:
    if(n<0):
        print("Invalid value of n")
        return -1
    if(n == 1 or n == 0):
        return 1
    return fibonacci(n-1) + fibonacci(n-2)

if __name__ == "__main__":
    print(fibonacci(5))